agent/agents/postgres/pgstatmonitor/pgstatmonitor.go

```diff
@@ -49,12 +49,14 @@ type PGStatMonitorQAN struct {
 	l                    *logrus.Entry
 	changes              chan agents.Change
 	monitorCache         *statMonitorCache
+	queryLength          int
```

Comment:
Why not `int32`?

Reply:
https://github.com/percona/pmm/pull/1216#discussion_r978350785